dali/operators/ssd/box_encoder.cc

```diff
@@ -188,33 +188,44 @@ DALI_REGISTER_OPERATOR(BoxEncoder, BoxEncoder<CPUBackend>, CPU);
 
 DALI_SCHEMA(BoxEncoder)
     .DocStr(
-        R"code(Encodes input bounding boxes and labels using set of default boxes (anchors) passed
-during op construction. Follows algorithm described in https://arxiv.org/abs/1512.02325 and
-implemented in https://github.com/mlperf/training/tree/master/single_stage_detector/ssd
-Inputs must be supplied as two Tensors: `BBoxes` containing bounding boxes represented as
-`[l,t,r,b]`, and `Labels` containing the corresponding label for each bounding box.
-Results are two tensors: `EncodedBBoxes` containing M encoded bounding boxes as `[l,t,r,b]`,
-where M is number of anchors and `EncodedLabels` containing the corresponding label for each
-encoded box.)code")
+        R"code(Encodes the input bounding boxes and labels using a set of default boxes (anchors)
+passed as an argument.
+
+This class follows the algorithm described in "SSD: Single Shot MultiBox Detector"
```

Comment:
Maybe `This operator` or something like that?

Reply:
Done